BoxOffice/Controller/DailyBoxOfficeViewController.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  DailyBoxOfficeViewController.swift
+//  BoxOffice
+//
+//  Created by kjs on 13/01/23.
+//  last modified by Idinaloq, MARY
+
+import UIKit
+
+final class DailyBoxOfficeViewController: UIViewController {
+    private var kobisOpenAPI: KobisOpenAPI = KobisOpenAPI()
+    private var networkService: NetworkService = NetworkService()
+    private var dateManager: DateManager = DateManager()
+    private var boxOfficeData: BoxOffice?
+    
+    private lazy var refresh: UIRefreshControl = {
+        let refreshControl: UIRefreshControl = UIRefreshControl()
+        refreshControl.addTarget(self, action: #selector(refreshCollectionView), for: .valueChanged)
+        
+        return refreshControl
+    }()
+    
+    let collectionView: UICollectionView = {
+        let layout: UICollectionViewFlowLayout = UICollectionViewFlowLayout()
+        let collectionView: UICollectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        layout.minimumLineSpacing = 0
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return collectionView
+    }()
+    
+    private lazy var activityIndicator: UIActivityIndicatorView = {
+        let activityIndicator: UIActivityIndicatorView = UIActivityIndicatorView()
+        activityIndicator.center = self.view.center
+        activityIndicator.color = .systemRed
+        activityIndicator.hidesWhenStopped = true
+        activityIndicator.style = UIActivityIndicatorView.Style.large
```

Comment:
```suggestion
        activityIndicator.color = .systemRed
        activityIndicator.hidesWhenStopped = true
        activityIndicator.style = .large
```
형식을 맞추면 더 보기 좋은 코드가 될 것 같습니다.

Reply:
해당 내용 반영하였습니다!